utils/glue_generator_src/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 cmake_minimum_required(VERSION 3.0.0)
 
-project(glue_generator LANGUAGES CXX VERSION 0.0.1)
+project(glue_generator VERSION 0.0.1)
```

Comment:
This is set by default, and it turns out that if you specify CXX (and not C) and try to compile with Visual Studio, then the C files are marked as not to compile. The easiest way is to omit this which sets the default to CXX and C.